widgetastic_patternfly.py

```diff
@@ -618,7 +618,7 @@ def select_by_visible_text(self, *items):
                 './div/ul/li/a[./span[contains(@class, "text") and normalize-space(.)={}]]'.format(
                     quote(text)),
                 parent=self)
-        self.close()
+        # self.close()
```

Comment:
![screenshot from 2017-03-13 09 12 48](https://cloud.githubusercontent.com/assets/3866976/23863521/46d16916-07cd-11e7-9db9-8e6ab7c3c1c3.png)

This is the first screen with Bootstrap element. It gets filled .